pkg/command/flag/usage.go

```diff
@@ -49,7 +49,7 @@ func postProcessUsage() {
 func cleanup(s string) string {
 	b := []byte(s)
 	b = bytes.ReplaceAll(b, []byte{'`', '`'}, []byte{'\''})
-	b = bytes.ReplaceAll(b, []byte{'\n', '*', ' '}, []byte{'\n'})
+	// b = bytes.ReplaceAll(b, []byte{'\n', '*', ' '}, []byte{'\n'})
```

Comment:
As @annastuchlik suggested, I added "starred" lists in all `sctool commands` - the result of this change is visible in the commit where `make docs` is performed.